python/samples/04.ai.a.teamsChefBot/pyproject.toml

```diff
@@ -0,0 +1,54 @@
+[tool.poetry]
+name = "teamschef-demo"
+version = "0.1.0"
+description = "Sample bot that thinks it's a Chef to help you cook Teams apps"
+authors = ["Microsoft"]
+readme = "README.md"
+packages = [
+    { include = "src" }
+]
+
+[tool.poetry.dependencies]
+python = "^3.8"
+fastapi = "^0.101.0"
+teams-ai = { path = "../../packages/ai", develop = true }
+uvicorn = "^0.23.2"
+python-dotenv = "^1.0.0"
+
+[tool.poetry.group.dev.dependencies]
+pytest = "^7.4.0"
+pylint = "^2.17.4"
+pytest-cov = "^4.1.0"
+pytest-asyncio = "^0.21.1"
+black = "^23.7.0"
+isort = "^5.12.0"
+mypy = "^1.5.0"
+
+[tool.poetry.scripts]
+lint = "scripts:lint"
+fmt = "scripts:fmt"
+test = "scripts:test"
+clean = "scripts:clean"
+ci = "scripts:ci"
+start = "scripts:start"
+
+[build-system]
+requires = ["poetry-core"]
+build-backend = "poetry.core.masonry.api"
+
+[tool.black]
+line-length = 100
+target-version = ['py38']
+
+[tool.isort]
+profile = "black"
+
+[tool.pytest.ini_options]
+addopts = "--cov-report html:coverage --cov=echo"
```

Comment:
update this to be `src` (I am fixing this in the other samples today)

Reply:
updated